### ISSUE TITLE
[Bug] Apps that autofill email data and attachments show as blank when opening ProtonMail

### ISSUE THREAD

**Issue by CorruptComputer** [ProtonMail/proton-mail-android#66]
**Describe the bug**
When an app tries to send an email with pre-filled information or attachments it will open ProtonMail and this info will be blank.
For this bug report I'll be using Signal as an example in the screenshots.

**To Reproduce**
1. Get an app to pre-fill data for an email.
2. Open in ProtonMail.
3. The email will be blank with no attachments.

**Expected behavior**
The data should be transferred and filled in.

**Screenshots**
Signal app before selecting email app to use:
![image](https://user-images.githubusercontent.com/5573038/106661602-624e3580-656f-11eb-8f50-72b1797f7f74.png)

ProtonMail app after Signal tries to open it:
![image](https://user-images.githubusercontent.com/5573038/106661741-91fd3d80-656f-11eb-8018-5a306cf00843.png)

Gmail app behavior for comparison:
![image](https://user-images.githubusercontent.com/5573038/106661841-b0633900-656f-11eb-96c4-cb13747a1189.png)

**Smartphone (please complete the following information):**
- Device: OnePlus 8T
- OS: Android 11 (Build number: Oxygen OS 11.0.6.8.KB05AA)
- Version: 1.13.24

**Additional context**
I contacted ProtonMail support about this, but all they said was "Our team is aware of the issue and we have passed the details of your case to our developers.". I checked the open issues here but couldn't find anything about it.

**Comment by NitroNils** [comment]
Yes, it has been a problem for many releases. Off the top of my head I can think of NewPipe and Barinstas bug reporting too getting blank fields.

**Comment by mtalexan** [comment]
Possibly related to #64?

**Comment by marinomeneghel** [comment]
Thanks for reporting this. 

We will raise this internally to assign it a priority, will come back here once we have a timeline for a solution.
